CODEOWNERS

```diff
@@ -5,7 +5,7 @@ kettle/** @rmmh
 logexporter/** @shyamjvs
 mungegithub/** @cjwagner
 planter/** @bentheelder
-prow/** @spxtr
+prow/** @cjwagner
```

Comment:
hey!

Reply:
Do you want to be added to all PRs still Joe?

Reply:
No that's fine :)